ketcher-autotests/tests/utils/clicks/selectButtonByTitle.ts

```diff
@@ -5,7 +5,10 @@ export async function selectButtonByTitle(title: ToolbarButton, page: Page) {
   await page.locator(`button[title*="${title}"]`).click();
 }
 
-// export async function getRotationHandleCoordinates(page: Page): Promise<Point> {
-//   // @ts-ignore
-//   return await page.evaluate(() => window.ketcher.editor.rotateController.handleCenter);
-// }
+export async function getRotationHandleCoordinates(page: Page) {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
Can we fix this warning?
Previously, we had two different repos and it wasn't easy to make changes simultaneously in both repos. But errors like that should be easily fixed now.